src/node/node_class.hpp

```diff
@@ -49,6 +49,8 @@ class ObjectWrap : public Nan::ObjectWrap {
         return Nan::New(js_template);
     }
 
+    static void construct(const v8::FunctionCallbackInfo<v8::Value>&);
```

Comment:
We should always use the v8 callback types over the Nan ones. We won't need to implement the callbacks for both v8 and Nan. Besides - there isn't an alias for `const v8::FunctionCallbackInfo<v8::Value>&` in Types.